all.js

```diff
@@ -25,9 +25,26 @@ function uploadFilesViaScp(sourceFileList, targetFileList, cb) {
     return;
   }
 
-  var prefix = config.device_user_name + ':' + config.device_password + '@' + config.device_host_name_or_ip_address + ':';
+  let scpOptions = {
```

Comment:
yes, definitely have tomake it consistent, i will leave it now, don't want to change this in hurry now